RollbarCommon/README.md

```diff
@@ -1,5 +1,5 @@
 # RollbarCommon
 
-This is an SDK module declaring protocols and implementing classes/types commonly used across the other SDK modules.
+This is an SDK module declaring abstractions/conepts (protocols, abstract classes, etc.) and implementing classes/types commonly used across the other SDK modules.
```

Comment:
NIt: typo "conepts" should be "concepts", I believe.

Reply:
@bishopb , thank you! fixing it...